README.md

```diff
@@ -1,8 +1,10 @@
+<!-- markdownlint-disable -->
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-17-orange.svg?style=flat-square)](#contributors-)
 
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- markdownlint -->
```

Comment:
Do we need both lines?
markdownlint-disable and markdonwlint?

Reply:
yes...

`<!-- markdownlint-disable -->` turns off the linter and it is not turned back on until `<!-- markdownlint -->` is passed.